app/Http/routes.php

```diff
@@ -42,6 +42,7 @@
     Route::get('messages/{message}/send', 'MessagesController@sendMessage')->name('messages.send');
 
     // Users
+    Route::get('users/contestants', 'UsersController@contestantIndex')->name('users.contestants');
```

Comment:
Yeah that's fine. You get a 404 cause it was trying to look up a user via `/users/:id` and using `contestant` as the id.